src/CascLib.h

```diff
@@ -316,6 +316,9 @@ typedef struct _CASC_OPEN_STORAGE_ARGS
 {
     size_t Size;                                // Length of this structure. Initialize to sizeof(CASC_OPEN_STORAGE_ARGS)
 
+    LPCTSTR szCdnHostUrl;                       // Online: Specify the cdn host url
+    LPCTSTR szCdnHostRegion;                    // Online: Specify the cdn host region
```

Comment:
For `ExtractVersionedArgument` to work properly, all new members must always be added at the end of the structure

Reply:
@Shauren Yep. That's why in my [incoming pull request](https://github.com/ladislav-zezula/CascLib/blob/53d417a21ce7529b65d4fae9dab3b3d360a7162d/src/CascLib.h#L344), it's been placed at the end of the structure.